stress/v2/statement/exec.go

```diff
@@ -0,0 +1,32 @@
+package statement
+
+import (
+	"time"
+
+	"github.com/influxdata/influxdb/stress/v2/ponyExpress"
+)
+
+// ExecStatement run outside scripts. This functionality is not built out
+// TODO: Wire up!
```

Comment:
Yes. This functionality was planned but has not been implemented. It is not mentioned in any of the documentation.